sphinx/source/governance.rst

```diff
@@ -0,0 +1,78 @@
+Governance
+==========
+
+A trusted set of members is in charge of governing a given CCF network. For transparency and auditability, all governance operations are recorded in plaintext in the ledger.
+
+One member (proposer) may submit a new proposal after which other members can vote for the proposal using its unique proposal ID. Proposals are executed once a quorum of members have accepted a proposal.
+
+The quorum is defined as a Lua script in the genesis transaction (see for example `the default quorum script`_). Common governance operations include adding a new user, member or a new version of the CCF code.
+
+.. note:: A proposal can be a Lua script defined by the proposer member or a static function defined by CCF (e.g. ``new_member``).
+
+.. _`the default quorum script`: https://github.com/microsoft/CCF/blob/master/src/runtime_config/gov.lua
+
+Submitting a new proposal
+-------------------------
+
+Assuming that 3 members (``member1``, ``member2`` and ``member3``) are already registered in the CCF network and that the quorum is defined as a strict majority of members, a member can submit a new proposal using the ``memberclient`` command-line utility (see :ref:`Member methods` for equivalent JSON-RPC API).
+
+For example, ``member1`` may submit a proposal to add a new member (``member4``) to the consortium:
+
+.. code-block:: bash
+
+    $ memberclient add_member --cert=member1_cert.pem --privk=member1_privk.pem --host=10.1.0.4 --port=25000 --ca=networkcert.pem --member_cert=member4_cert.pem
+    {"commit":100,"global_commit":99,"id":0,"jsonrpc":"2.0","result":{"completed":false,"id":1},"term":2}
+
+In this case, the proposal has successfully been created and given the proposal id ``1``. Other members, including ``member1``, can then accept or reject the proposal:
+
+.. code-block:: bash
+
+    // Member 1 accepts the proposal
```

Comment:
We assume that the member that makes the proposal also accepts it, so there is no need for the member who made the proposal to vote for it. This could be added in the documentation.

Reply:
Good point! What threw me is that the vote of the proposer is not displayed when displaying the proposals but the proposer can still vote for it (with no effect of course).